examples/clickpipe/object_storage_s3_sqs_iam_role/provider.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    clickhouse = {
+      source = "ClickHouse/clickhouse"
```

Comment:
let's please use the template like we do here: https://github.com/ClickHouse/terraform-provider-clickhouse/blob/main/examples/clickpipe/externally_managed_table/provider.tf.template.alpha